pkgs/applications/misc/clipmenu/default.nix

```diff
@@ -4,21 +4,23 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "clipmenu";
-  version = "6.0.1";
+  version = "6.1.0";
 
   src = fetchFromGitHub {
     owner  = "cdown";
     repo   = "clipmenu";
     rev    = version;
-    sha256 = "0053j4i14lz5m2bzc5sch5id5ilr1bl196mp8fp0q8x74w3vavs9";
+    sha256 = "0ddj5xcwrdb2qvrndvhv8j6swcqc8dvv5i00pqk35rfk5mrl4hwv";
   };
 
+  postPatch = ''
+    substituteInPlace ./Makefile --replace /usr "$out"
```

Comment:
You can use fetchpatch with `https://github.com/cdown/clipmenu/commit/443b58583ef216e2405e4a38d401f7c36386d21e.patch` now.
I added it in this PR: https://github.com/cdown/clipmenu/pull/135

Reply:
I've never used `fetchpatch` before, I hope this is the right way to do it...